README.md

```diff
@@ -36,10 +36,7 @@ Node 8.9.x and NPM 5.6.x (and above) are **required**. We suggest using [nvm](ht
 
 3. Add developer files for config overrides:
 
-  ```sh
-  npm run scaffold
-  ```
-  This will create `webpack/proxy.dev.js` and `src/js/config/Config.dev.ts`. These files aren't tracked by git and provide a place to override proxy and general dcos-ui configuration.
+  Create `webpack/proxy.dev.js` and `src/js/config/Config.dev.ts` from provided template files in same directory. These files aren't tracked by git and provide a place to override proxy and general dcos-ui configuration.
```

Comment:
They are still written, aren't they?

Reply:
forgot to remove it after adding `scaffold` back for CI…